### PR TITLE
[bitnami/elasticsearch] Unify and document production values

### PR DESCRIPTION
**Pull request by carrodher**

### CODE REVIEW

File: bitnami/elasticsearch/README.md [hunk 1]
```diff
@@ -197,6 +197,174 @@ $ helm install --name my-release -f values.yaml bitnami/elasticsearch
 
 > **Tip**: You can use the default [values.yaml](values.yaml). [values-production.yaml](values-production.yaml) has defaults optimized for use in production environments.
 
+### Production configuration
```
Comment (juan131) [inline bitnami/elasticsearch/README.md]: I'd update the "Tip" note with a link to this section. I mean this one:

```
> **Tip**: You can use the default [values.yaml](values.yaml). [values-production.yaml](values-production.yaml) has defaults optimized for use in production environments.
```

File: bitnami/elasticsearch/values-production.yaml [hunk 2]
```diff
@@ -117,16 +125,11 @@ securityContext:
...
-##
-discovery:
-  name: discovery
-
 ## Elasticsearch coordinating-only node parameters
 ##
 coordinating:
   name: coordinating-only
-  replicas: 2
+  replicas: 3
```
Comment (juan131) [inline bitnami/elasticsearch/values-production.yaml]: Why this change?

File: bitnami/elasticsearch/values-production.yaml [hunk 3]
```diff
@@ -243,7 +251,7 @@ data:
 ingest:
   enabled: true
   name: ingest
-  replicas: 2
+  replicas: 3
```
Comment (juan131) [inline bitnami/elasticsearch/values-production.yaml]: Why this change?

File: bitnami/elasticsearch/values.yaml [hunk 4]
```diff
@@ -18,7 +18,7 @@ image:
   ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
   ##
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```
Comment (juan131) [inline bitnami/elasticsearch/values.yaml]: Why did you change the `pullPolicy` in the **values.yaml**?

File: bitnami/elasticsearch/values.yaml [hunk 5]
```diff
@@ -310,7 +307,7 @@ metrics:
     registry: docker.io
     repository: bitnami/elasticsearch-exporter
     tag: 1.0.2
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```
Comment (juan131) [inline bitnami/elasticsearch/values.yaml]: Why did you change the `pullPolicy` in the **values.yaml**?

Comment (carrodher) [reply bitnami/elasticsearch/values.yaml]: Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images

File: bitnami/elasticsearch/values.yaml (hunk 4 above)
Comment (carrodher) [reply bitnami/elasticsearch/values.yaml]: Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images

File: bitnami/elasticsearch/values-production.yaml (hunk 2 above)
Comment (carrodher) [reply bitnami/elasticsearch/values-production.yaml]: We are increasing the number of nodes from 2 to 3 in the case of `master` and `data`, why not for `coordinating` and `ingest`?

File: bitnami/elasticsearch/values-production.yaml (hunk 3 above)
Comment (carrodher) [reply bitnami/elasticsearch/values-production.yaml]: We are increasing the number of nodes from 2 to 3 in the case of `master` and `data`, why not for `coordinating` and `ingest`?
